pkg/agent/sysadvisor/plugin/poweraware/evictor/server/evictor_service.go

```diff
@@ -43,24 +42,47 @@ const (
 var errPowerPressureEvictionPluginUnavailable = errors.New("power pressure eviction plugin is unavailable")
 
 type powerPressureEvictServer struct {
+	plugin  *powerPressureEvictPlugin
```

Comment:
Where is `plugin` initialized?

Reply:
thanks for catching this - will add it as server object field